contracts/state/State.sol

```diff
@@ -92,11 +92,12 @@ contract State is Ownable2StepUpgradeable, IState {
     }
 
     /**
-     * @dev Set defaultIdType external wrapper (only owner can call)
-     * @param defaultIdType default id type
+     * @dev Get defaultIdType
+     * @return defaultIdType
      */
-    function setDefaultIdType(bytes2 defaultIdType) external onlyOwner {
```

Comment:
We still need this function (setDefaultIdType) and we'll call it on polygon mainnet & amoy.